runtime/plugin/osc52.lua

```diff
@@ -0,0 +1,26 @@
+if
+  vim.g.osc52 == false
+  or (
+    vim.g.osc52 == nil and (not os.getenv('SSH_TTY') or os.getenv('TMUX') or vim.o.clipboard ~= '')
+  )
+then
+  return
+end
+
+require('vim.termcap').query('Ms', function(cap, seq)
```

Comment:
nit: query is a good name but would `request` make sense (for symmetry with [TermRequest](https://github.com/neovim/neovim/pull/22159))? Don't want a "foolish consistency", but _if_ they are truly equivalent then using common language is good.